docs/en_US/Release_v1.0.md

```diff
@@ -0,0 +1,33 @@
+
+<p align="center">
+<img src=".././img/release-1-title-1.png" width="100%" />
+</p>
+
+
+From September, 2018 to September, 2019, We are still moving on …
```

Comment:
I think September 2018 without comma is more appropriate

Reply:
ok, fixed~